scripts/plotting/zonal_mean.py

```diff
@@ -269,6 +252,16 @@ def zonal_mean(adfobj):
                         plot_name.unlink()
                     #End if
 
+                    #Seasonal Averages
+                    mseasons[s] = pf.seasonal_mean(mdata, season=s, is_climo=True) #mdata.sel(time=seasons[s]).mean(dim='time')
+                    oseasons[s] = pf.seasonal_mean(mdata, season=s, is_climo=True) #odata.sel(time=seasons[s]).mean(dim='time')
```

Comment:
I think we'll want to use `odata` here, and get rid of the commented out code:

```suggestion
                    mseasons[s] = pf.seasonal_mean(mdata, season=s, is_climo=True)
                    oseasons[s] = pf.seasonal_mean(odata, season=s, is_climo=True)
```